package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "hb-multipage-site",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "start": "webpack-dev-server"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/milenacard/hb-multipage-site.git"
+  },
+  "author": "Milena Cardenas",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/milenacard/hb-multipage-site/issues"
+  },
+  "homepage": "https://github.com/milenacard/hb-multipage-site#readme",
+  "devDependencies": {
+    "css-loader": "^0.28.9",
+    "extract-text-webpack-plugin": "^3.0.2",
+    "html-webpack-plugin": "^2.30.1",
+    "husky": "^0.14.3",
```

Comment:
It's fine if you want to use `husky` but I don't see any script that is actually going to use it.